openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/index.tsx

```diff
@@ -11,31 +11,29 @@
  *  limitations under the License.
  */
 
+import { Modal, Typography } from 'antd';
 import classNames from 'classnames';
 import { CookieStorage } from 'cookie-storage';
+import { t } from 'i18next';
 import React, { FunctionComponent, useEffect, useState } from 'react';
+import CloseWhatNewIconComponent from '../CloseWhatNewIconComponent';
 import ChangeLogs from './ChangeLogs';
 import FeaturesCarousel from './FeaturesCarousel';
 import { COOKIE_VERSION, LATEST_VERSION_ID, WHATS_NEW } from './whatsNewData';
+import { ToggleType, WhatsNewModalProps } from './WhatsNewModal.interface';
+import './WhatsNewModal.styles.less';
 import { getReleaseVersionExpiry } from './WhatsNewModal.util';
-// import { Button } from '../../buttons/Button/Button';
-
-type Props = {
-  header: string;
-  onCancel: () => void;
-};
-
-type ToggleType = 'features' | 'change-log';
 
 const iconString = `M7.878 3.002a1.876 1.876 0 11-3.751 0 1.876 1.876 0 013.751 0zm1.073.562a3.003
```

Comment:
Any reason we did this? If not can we eliminate creating a variable for this?

Reply:
It was already there, not a part of this PR.

Reply:
Sure, I got that. I was just asking to check if we can eliminate that as we are already touching the code.

Reply:
Sure, will check.